.github/workflows/test.yml

```diff
@@ -22,4 +22,15 @@ jobs:
           key: v1/${{ runner.os }}/node-${{ matrix.node }}/${{ hashFiles('**/yarn.lock') }}
           restore-keys: v1/${{ runner.os }}/node-${{ matrix.node }}/
       - run: yarn
+      - name: Set up @percy/cli master
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |
+          git clone https://github.com/percy/cli
+          cd cli
+          yarn
+          yarn build
+          yarn global:link
+          cd .. 
+          yarn remove @percy/cli && yarn link @percy/cli
+          npx percy --version
```

Comment:
```suggestion
```

* cloning the version won't help, since we're ideally not bumping the version
* what you could do is log the `commit` and `commit_msg`

Reply:
This is for confirming that we are using the version from git in `npx percy`. Commit would be the latest since we create a fresh clone, so checking it will give the latest. If there's some way that npx percy would give the commit id, it'd be better to use it, otherwise we can ignore

Reply:
I'd suggest removing, since the version information won't be very useful here.